Tools/Scripts/webkitpy/llvm_profile_utils.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+import glob
+import logging
+import math
+import os
+import shlex
+import subprocess
+
+from functools import cache
+
+logger = logging.getLogger(__name__)
+
+
+def locate_xcrun_binary(sdk, binary_name):
```

Comment:
Maybe `which_in_sdk` or something? The name of the function makes it sound more like we're finding `xcrun` itself.

Reply:
`which_in_sdk` might be a bit misleading, I would go with `locate_binary_via_xcrun`